doc/source/limitation.rst

```diff
@@ -6,30 +6,32 @@ Percona XtraDB Cluster Limitations
 
 The following limitations apply to |PXC|:
 
-* Replication works only with |InnoDB| storage engine.
-  Any writes to tables of other types, including system (``mysql.*``) tables,
-  are not replicated.
-  However, ``DDL`` statements are replicated in statement level,
-  and changes to ``mysql.*`` tables will get replicated that way.
-  So you can safely issue ``CREATE USER...``,
-  but issuing ``INSERT INTO mysql.user...`` will not be replicated.
-  You can enable experimental |MyISAM| replication support
-  using the :variable:`wsrep_replicate_myisam` variable.
+Replication works only with |InnoDB| storage engine.
+   Any writes to tables of other types, including system (``mysql.*``)
+   tables, are not replicated.
 
-* Unsupported queries:
+   However, ``DDL`` statements are replicated in statement level, and
+   changes to ``mysql.*`` tables will get replicated that way.  So you
+   can safely issue ``CREATE USER...``, but issuing ``INSERT INTO
+   mysql.user...`` will not be replicated.  You can enable
```

Comment:
I don't think we should suggest MyISAM replication.  It's experimental in Galera 3, I'm not sure if it's even still in Galera 4 considering MyISAM is being deprecated.